src/server/common/controller/page-controller/page-controller.js

```diff
@@ -0,0 +1,61 @@
+import { calculateNextPage } from '../../helpers/next-page.js'
+import { ExitPage } from '../../model/page/exit-page-model.js'
+/** @import { Server, ServerRegisterPluginObject } from '@hapi/hapi' */
+/** @import { NextPage } from '../../helpers/next-page.js' */
+/** @import { RawPayload } from '../../model/answer/answer-model.js' */
+/** @import { Page } from '../../model/page/page-model.js' */
+
+export class PageController {
+  /**
+   * @param {Page} page
+   */
+  constructor(page) {
+    this.page = page
+  }
+
+  /** @returns {ServerRegisterPluginObject<void>} */
+  plugin() {
+    return {
+      plugin: {
+        name: `${this.page.sectionKey}-${this.page.key}`,
+
+        /** @param {Server} server */
+        register: (server) => {
+          server.route([
+            {
+              method: 'GET',
+              path: this.page.urlPath,
+              handler: this.getHandler.bind(this)
+            },
+            {
+              method: 'POST',
+              path: this.page.urlPath,
+              handler: this.postHandler.bind(this)
+            }
+          ])
+        }
+      }
+    }
+  }
+
+  getHandler(req, h) {
+    return h.view(this.page.view, {
+      nextPage: req.query.redirect_uri,
+      pageTitle: this.page.title,
+      heading: this.page.heading,
+      hideQuestion: true,
```

Comment:
I don't mind merging like this, but it makes more sense (to me) to have a separate non-question content layout for generic pages.

Reply:
I think if it gets any more complicated i would agree, but this is purely to hide the default "continue button" that is always present really

the other way was to override the block with an empty slot